pkg/policy/api/fqdn.go

```diff
@@ -83,3 +83,14 @@ func (r *PortRuleDNS) Sanitize() error {
 	}
 	return matchpattern.Validate(r.MatchPattern)
 }
+
+func (s *FQDNSelector) GetDestinationEndpointSelectors() EndpointSelectorSlice {
```

Comment:
This function isn't used anywhere.

Reply:
yeah, I had started mimicking other rule types, but then ended up returning a single selector for the whole list (other types' `fooList.GetDestinationEndpointSelectors` iterated over the list and called `GetDestinationEndpointSelectors` on each).
I'll clean this up regardless.